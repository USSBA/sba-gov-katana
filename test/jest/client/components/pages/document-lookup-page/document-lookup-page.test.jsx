/*global expect*/

/* eslint-disable no-unused-vars,no-undef */
import React from "react";
/*eslint-enable no-unused-vars*/
/*
import DocumentLookupPage from "client/components/pages/document-lookup-page/document-lookup-page.jsx";
import ShallowRenderer from 'react-test-renderer/shallow';
import renderer from "react-test-renderer";
import _ from "lodash";
*/
describe("DocumentLookupPage", () => {
  test("renders a page lookup with the correct attributes", () => {
    /*
    const renderer = new ShallowRenderer();
    const component = renderer.render(<DocumentLookupPage/>);
    expect(component).toMatchSnapshot();
    */
  });
});
