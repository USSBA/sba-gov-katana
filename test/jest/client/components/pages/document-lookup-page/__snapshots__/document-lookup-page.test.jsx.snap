// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentLookupPage renders a page lookup with the correct attributes 1`] = `
<Connect(PagingLookup)
  fieldsToShowInDetails={
    Array [
      "Activity",
      "Program",
      "Summary",
    ]
  }
  sortByOptions={
    Array [
      "Last Updated",
      "Title",
      "Number",
    ]
  }
  taxonomyFilters={
    Array [
      "documentType",
      "program",
      "documentActivity",
    ]
  }
  title="Documentation Lookup"
  type="documents"
/>
`;
